frontend/src/components/layouts/AppLayout.tsx

```diff
@@ -167,6 +168,18 @@ export default function AppLayout() {
           <Megaphone className="h-4 w-4" />
           Feedback
         </MenuItem>
+        <MenuItem
+          to="/"
+          onClick={(e) => {
+            openLink(
+              "https://guides.getalby.com/user-guide/v/alby-account-and-browser-extension/alby-hub"
+            );
+            e.preventDefault();
+          }}
+        >
+          <Lightbulb className="h-4 w-4" />
+          Alby Guides
```

Comment:
Not sure about the title, it feels wrong to add "Alby" here. 

What about "Knowledge Base" or "Help center"?

Reply:
will fix in followup PR, thanks!